bin/cli.js

```diff
@@ -133,6 +144,35 @@ program.command('create')
       }
     }
 
+    // Ingest user-provided CDX files, if any.
+    if (values?.cdxj) {
+      try {
+        const dirPath = values?.cdxj
+        const cdxjFiles = await fs.readdir(dirPath)
+        const allowedExts = ['cdx', 'cdxj']
+
+        for (let i = 0; i < cdxjFiles.length; i++) {
+          const cdxjFile = resolve(dirPath, cdxjFiles[i])
+
+          const ext = cdxjFile.split('.').pop()
+          if (!allowedExts.includes(ext)) {
+            log.info(`CDXJ: Skipping file ${cdxjFile}, not a CDXJ file`)
```

Comment:
@tw4l Nitpick: I'd make that a warning maybe?

Reply:
Good call, just pushed the change :)